core/src/main/resources/schema/infinispan-config-8.1.xsd

```diff
@@ -1160,15 +1160,28 @@
             </xs:documentation>
           </xs:annotation>
         </xs:attribute>
-         <xs:attribute name="consistent-hash-factory" type="xs:string">
-            <xs:annotation>
-               <xs:documentation>
-                  The factory to use for generating the consistent hash. Must implement org.infinispan.distribution.ch.ConsistentHashFactory.
-                  This option can be used to guarantee that multiple distributed caches use exactly the same consistent hash,
-                  which for performance reasons is not guaranteed by the default consistent hash factory instance used.
-               </xs:documentation>
-            </xs:annotation>
-         </xs:attribute>
+        <xs:attribute name="consistent-hash-factory" type="xs:string">
+          <xs:annotation>
+            <xs:documentation>
+              The factory to use for generating the consistent hash.
```

Comment:
Except it's not :)
The hash function that I've deprecated was never exposed in the XML configuration.